src/main/java/org/springframework/samples/petclinic/mapper/PetMapper.java

```diff
@@ -17,13 +17,15 @@
 public interface PetMapper {
 
     @Mapping(source = "owner.id", target = "ownerId")
+    @Mapping(source = "weight", target = "weight")
```

Comment:
Why is this needed? 

I think the mapping annotation above is only present there, because it's a transitive field, but I don't think mapping for a normal field is needed.

(same below, of course)